cli/azd/cmd/middleware/telemetry.go

```diff
@@ -89,171 +77,8 @@ func (m *TelemetryMiddleware) Run(ctx context.Context, next NextFn) (*actions.Ac
 	result.TraceID = span.SpanContext().TraceID().String()
 
 	if err != nil {
-		mapError(err, span)
+		cmd.MapError(err, span)
 	}
 
 	return result, err
 }
-
```

Comment:
This was a move-refactor to reshare mapping of application errors, see [commit](https://github.com/Azure/azure-dev/pull/3453/commits/7c4d5323af518c8318ac7384c32821c7c3d7033f). Also happy to split this into it's own PR if requested.